pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/module/ModuleVersionUpload.java

```diff
@@ -73,10 +73,7 @@ protected void doRun(SlingHttpServletRequest request, PostResponse response, Lis
 
         try {
             String locale = ServletUtils.paramValue(request, "locale", GlobalConfig.DEFAULT_MODULE_LOCALE.toString());
-            String encoding = request.getCharacterEncoding();
-            if (encoding == null) {
-                encoding = StandardCharsets.UTF_8.name();
-            }
+            String encoding = StandardCharsets.UTF_8.name();
```

Comment:
(same comment as above)

Reply:
@benradey if you check encoding, after running uploader script, you will find, it's always ISO_8859_1, There is no use of this conditional block with current code, as it will never execute.

secondly, if you check com.redhat.pantheon.servlet.ServletUtils#handleParamAsStream(org.apache.sling.api.SlingHttpServletRequest, java.lang.String, java.util.function.Function<java.io.InputStream,R>) , line no 184, you can see, we are parsing content in UTF_8 while storing it in nodes.

Accordingly, i understand, we can go ahead by removing this block, to fix the encoding issue.

Reply:
@rednitish Same, please remove this variable and all subsequent references.